assets/js/blocks/attribute-filter/edit.js

```diff
@@ -258,45 +258,42 @@ const Edit = ( { attributes, setAttributes, debouncedSpeak } ) => {
 		</Placeholder>
 	);
 
-	const onDone = useCallback( () => {
+	const onDone = () => {
```

Comment:
I don't think it made much sense wrapping this function in a `useCallback()` because it's not used inside any other hook.